storage/memory/memory.go

```diff
@@ -226,6 +227,7 @@ func (m *memory) RemoveTriples(ctx context.Context, ts []*triple.Triple) error {
 // considered on a certain operation.
 type checker struct {
 	max bool
+	m   int
 	c   int
```

Comment:
Mind further elaborating what is the semantics of these fields?

Reply:
Done